base/condition.jl

```diff
@@ -162,3 +162,8 @@ this, and can be used for level-triggered events.
 This object is NOT thread-safe. See [`Threads.Condition`](@ref) for a thread-safe version.
 """
 const Condition = GenericCondition{AlwaysLockedST}
+
+lock(c::GenericCondition{AlwaysLockedST}) =
+    throw(ArgumentError("`Condition` is NOT thread-safe. Please use `Threads.Condition` instead for concurrent code."))
```

Comment:
```suggestion
    throw(ArgumentError("`Condition` is not thread-safe. Please use `Threads.Condition` instead for multi-threaded code."))
```

Reply:
Thanks Jeff, applied your change. I originally copied that emphasis from the docstring, which says:
```
This object is NOT thread-safe. See [`Threads.Condition`](@ref) for a thread-safe version.
```
haha :) But i agree the lower-case is better